src/App.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useReducer } from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { Button, StyleSheet, Text, View } from 'react-native';
 import AddItem from './components/AddItem';
 import AddPartyMember from './components/AddPartyMember';
 import Tax from './components/Tax';
 import Tip from './components/Tip';
 import { DispatchActions } from './constants';
 import { DispatchActionArgs, AppState } from './types';
 
+const APP_CONTAINER_COLOR = '#fff';
```

Comment:
I had to create a const for this color weirdly since I was getting a react-native eslint error regarding 
```
Color literal: { backgroundColor: '#fff' }                      
react-native/no-color-literals
```

not sure if this should go in the constants file though... or how we want to store styles for each component in general?

Reply:
Having the styles in a central place like a styles folder/file makes sense. We can probably do this when we are implementing the design. :D